core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/HealthMonitor.scala

```diff
@@ -242,25 +290,12 @@ class HealthMonitor private (val slickDataSource: SlickDataSource,
   }
 
   /**
-    * Checks Google groups status by doing a Get on the admin and curator groups using the groups
-    * service account.
-    */
-  private def checkGoogleGroups: Future[SubsystemStatus] =
```

Comment:
I deleted all traces of this check. It was already gutted and rawls doesn't depend (directly) on google groups anyway.